api/src/main/java/marquez/db/JobDao.java

```diff
@@ -78,16 +78,24 @@
   }
 
   @SqlQuery(
-      "SELECT j.*, n.name AS namespace_name FROM jobs AS j "
-          + "INNER JOIN namespaces AS n "
-          + "  ON (n.name = :namespaceName AND "
-          + "      j.namespace_uuid = n.uuid AND "
-          + "      j.name = :jobName)")
+      """
+          SELECT j.* FROM jobs_view AS j
+          WHERE j.namespace_name = :namespaceName AND
+          j.name = :jobName
+      """)
   Optional<JobRow> findJobByNameAsRow(String namespaceName, String jobName);
```

Comment:
Does this query return the FQN of the job? I'm just trying to understanding the difference between `findJobByNameAsRow()` and `findJobBySimpleNameAsRow()` when using the view vs not.

Reply:
I see the `job_fqn` function defined in the migration `V41__alter_jobs_add_job_parent_id.sql` constructs the fully qualified name.